casepro/utils/tests.py

```diff
@@ -87,6 +92,23 @@ def test_month_range(self):
         self.assertEqual(month_range(-1, now=d1), (datetime(2015, 9, 1, 0, 0, 0, 0, pytz.UTC),
                                                    datetime(2015, 10, 1, 0, 0, 0, 0, pytz.UTC)))
 
+    def test_uuid_to_int_range(self):
+        '''Ensures that the integer returned will always be in the range
+        [-2147483648, 2147483647].'''
+        self.assertEqual(
+            uuid_to_int(UUID(int=(2147483648 + 2147483647)).hex),
+            2147483647)
+        self.assertEqual(
+            uuid_to_int(UUID(int=(2147483648 + 2147483647 + 1)).hex),
+            -2147483648)
+
+    @given(st.uuids())
+    def test_uuid_to_int_property(self, uuid):
+        '''Property based testing to ensure that the output of the function
+        is always within the limits.'''
+        self.assertTrue(uuid_to_int(uuid.hex) <= 2147483647)
+        self.assertTrue(uuid_to_int(uuid.hex) >= -2147483648)
```

Comment:
Possible, but since this is random uuids, I wanted a test with specific values to test the edges.